src/components/Events.vue

```diff
@@ -9,43 +9,14 @@
 </template>
 
 <script>
-import axios from "@nextcloud/axios";
```

Comment:
I don't think all this should be removed, it seems to break the upcoming events

Reply:
It was because I did not pass props proberly.

Reply:
Fixed in last commit